src/Nethermind/Nethermind.JsonRpc/Modules/Parity/IParityModule.cs

```diff
@@ -24,11 +24,21 @@ namespace Nethermind.JsonRpc.Modules.Parity
     [RpcModule(ModuleType.Parity)]
     public interface IParityModule : IModule
     {
+        [JsonRpcMethod(Description = "Returns a list of transactions currently in the queue.", Returns = "Array", IsImplemented = true)]
         ResultWrapper<ParityTransaction[]> parity_pendingTransactions();
+
+        [JsonRpcMethod(Description = "Get receipts from all transactions from particular block, more efficient than fetching the receipts one-by-one.", Returns = "Array", IsImplemented = true)]
         ResultWrapper<ReceiptForRpc[]> parity_getBlockReceipts(BlockParameter blockParameter);
+
+        [JsonRpcMethod(Description = "Returns the node enode URI.", Returns = "String", IsImplemented = true)]
```

Comment:
enode is something different than uri - it is a combination of public key and uri